app/views/market_places/index.html.erb

```diff
@@ -15,73 +15,48 @@
+              </svg>            
+            </div>
           </button>
         </div>
         <div class="flex-initial px-2">
-          <button name="business_line" type="submit" value="All" class=""><%= image_tag 'md/food.svg', class: 'm-auto' %>
-            <p>Todos</p>
+          <button name="business_line" type="submit" value="All">
+            <div class='hover:text-tribo focus:text-tribo'>
+              <svg class='fill-current' height="41" viewBox="0 0 38 41" xmlns="http://www.w3.org/2000/svg">
```

Comment:
WDYT about moving this SVG to the `assets`?

Reply:
We put them directly in the html, because we used a class in the svg, to change the fill color, since we couldn't do it by calling it from an <img> tag.

Reply:
I think it's possible. You can use selectors. But not pretty sure tbh
Perhaps @jaimelr or @oswaldotovar32 Can help you to figure out how to style svgs.

Another alternative is moving them into partials. 
IMO it makes it a bit hard to read the HTML

Reply:
@EduardoGHdez Looking for options to solve the problem we found we don’t get to control the "innards" of the SVG with CSS.